feat: @sap/cds version updated to 5.7.1

Bonny88 opened the pull request:
Hi @mikezaschka 

I have updated the version of the package @sap/cds to the latest available (5.7.1) in order to fix the issue caused by the missing object @sap/cds/bin/build/buildTaskHandlerFactory when launching the deploy command.

The new version seem to work as expected. I've tried it by compiling the solution locally and running the cli.js script with the arguments "deploy -c".

I didn't update the changelog or the package version though.
Let me know if it's ok or you have any issues.

Thanks for your work on this project
Bonny

gregorwolf commented:
Hi @Bonny88,

thank you for this contribution. Before I merge yours I've merged #98 to have the upgrade to node 14 in. Can you rebase and update your PR?

CU
Gregor

Bonny88 commented:
Hi @gregorwolf.

I've rebased and updated the PR. I recreated the package-lock.json after the merge to ensure the consistency with the updated package.json.

Thanks again
Bonny
